server/edd/account.py

```diff
@@ -181,3 +182,20 @@ def save(self, request, **kwargs):
             self.users = account.utils.filter_users_by_email(email)
         # base class .save() call generates reset token and sends email to self.users
         return super().save(request, **kwargs)
+
+
+class SignupForm(BaseSignupForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["password1"].help_text = "Must be at least 6 characters in length"
```

Comment:
This should be sourced from current configured password validator(s), and not hard-coded. If user-facing text in a form _is_ hard-coded, it should also be wrapped in `_()` aka `gettext()` or `gettext_lazy()`.

Reply:
I'll see how involved it is to get that info, and worst case I'll replace this with a localization call as mentioned.

Reply:
Should be able to use [password_validators_help_texts](https://docs.djangoproject.com/en/4.0/topics/auth/passwords/#django.contrib.auth.password_validation.password_validators_help_texts) for this, though I have not yet tried to use this.

Reply:
I couldn't get that work (probably because of [how the message is defined](https://github.com/pennersr/django-allauth/blob/db1cfb241a7b1bc27e9a1bd81eaca5880861b3b6/allauth/account/adapter.py#L301-L305)), but I was able to use similar code to localize the requirements text, including getting the exact number of characters required.